packages/core/src/types/select.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface ISelectOption {
```

Comment:
I would rename this to `WayAutosuggestOption`. The `I` is not a TypeScript convention.